src/main/java/net/fabricmc/loader/discovery/ModResolver.java

```diff
@@ -696,6 +713,12 @@ protected void compute() {
 									}
 
 									list.add(dest);
+
+									try {
+										readableNestedJarPaths.put(UrlUtil.asUrl(dest).toString(), String.format("%s!%s", getReadablePath(loader, candidate), modPath));
+									} catch (UrlConversionException e) {
+										e.printStackTrace();
```

Comment:
How about logging this instead?

Reply:
True, that would probably lead to more readable error messages.